README.rst

```diff
@@ -0,0 +1,73 @@
+Django Preview Image Widget
+===========================
+
+    An ImageField preview widget application
```

Comment:
This can be changed to A very simple ImageField preview widget to highlight the simplicity of the widget.